gitlab/base.py

```diff
@@ -38,7 +40,7 @@ class RESTObject(object):
 
     _id_attr = "id"
 
-    def __init__(self, manager, attrs):
+    def __init__(self, manager: "RESTManager", attrs: Dict[str, Any]) -> None:
```

Comment:
Sorry just a quick question - the string literals are here for python 3.6 only (e.g. https://stackoverflow.com/q/33533148)?

Reply:
Yes, because we can't use:

```
from __future__ import annotations
```

Reply:
Ok, thanks! Yeah, seems like it might still be an issue with objects returning self anyway.